site/docs/0.0/content/font.md

```diff
@@ -0,0 +1,746 @@
+- Workflow challenges with making updates to centrally managed web fonts.
+
+<!--Using shared standard assets is inherently faster across multiple-->
+<!--domains, any deviation to using shared standard assets should be considered-->
+<!--carefully by researching traffic flow between institutional web properties, and-->
+<!--not just initial page load times on one site.  In other words all sites that-->
+<!--link to arizona.edu domain, typically share users with the parent domain, and-->
+<!--other subdomains. Browsers cache assets based on the address it is downloaded-->
+<!--from, so if all subdomains share the same asset, the end user's browser will-->
+<!--access a cached copy without downloading it again across those subdomains.-->
+
+
+## Global settings
+For projects not using Arizona Bootstrap, follow these instructions on how to add Proxima Nova to your website. Be aware that Arizona Bootstrap reference links come "pre-loaded" with the Proxima Nova font suite. In other words, if you're using Arizona Bootstrap, font installation instructions are not required.
+
+
+### **Native Fallback Fonts**
+The fallback web fonts (Verdana, Geneva) have been dropped in Arizona Bootstrap
+4 and replaced with a “native font stack” for optimum text rendering on every
+device and OS. Read more about
+[native font stacks in this Smashing Magazine article](https://www.smashingmagazine.com/2015/11/using-system-ui-fonts-practical-guide/).
+
+```
+$font-family-sans-serif:
+  proxima-nova,
+  // Safari for macOS and iOS (San Francisco)
+  -apple-system,
+  // Chrome < 56 for macOS (San Francisco)
+  BlinkMacSystemFont,
+  // Windows
+  "Segoe UI",
+  // Android
+  "Roboto",
+  // Basic web fallback
+  "Helvetica Neue", Arial, sans-serif,
+  // Emoji fonts
+  "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol" !default;
+```
+  This `font-family` is applied to the `<body>` and automatically inherited
+  globally throughout Bootstrap. To switch the global `font-family`, update
+  `$font-family-base` and recompile Arizona Bootstrap.
+
+## How to Use
+  The Milo font suite is available for official use by UA employees through
+  a license with FontShop. Please follow the instructions below to ensure
+  compliance with the license.
+
+### **Instructions**
+
+  Before using the Milo font suite, please complete the <a href="https://brand.arizona.edu/font-license-agreement" target="_blank">font license
+  agreement</a> <span class="label
+  label-warning">Important</span>
+
+  For projects not using UA Bootstrap, follow these instructions on <a href="https://brand.arizona.edu/how-add-milo-your-website" target="_blank">how to add Milo to your website</a>.
+  Be aware that [UA Bootstrap reference links](index.html#ua-bootstrap-reference-links) come "pre-loaded"
+  with the Milo font suite. In other words, if you're using UA Bootstrap, font
+  installation instructions are not required.<span class="label
+  label-warning">Important</span>
```

Comment:
Do we really still want to reference Milo / UA Bootstrap here?

Reply:
Looks like I left a lot of stuff in there on accident

Reply:
Removed all of that.